geode-core/src/main/java/org/apache/geode/internal/cache/wan/AbstractGatewaySender.java

```diff
@@ -1034,10 +1034,7 @@ public void distribute(EnumListenerEvent operation, EntryEventImpl event,
       // If this gateway is not running, return
       if (!isRunning()) {
         if (this.isPrimary()) {
-          tmpDroppedEvents.add(clonedEvent);
-          if (isDebugEnabled) {
-            logger.debug("add to tmpDroppedEvents for evnet {}", clonedEvent);
-          }
+          dropEvent(clonedEvent);
```

Comment:
this should be recordDroppedEvent or similar name.

Reply:
I agree. Thanks